dynamic_programming/climbing_stairs.py

```diff
@@ -1,27 +1,42 @@
+#!/usr/bin/env python3
```

Comment:
LGTM. The shebang line can be removed btw. Thank you!

Reply:
On Macs and Linux the shebang line enables an effective shorthand.  It’s presence allows us to execute the script with __./climb*__  so why do we want to type more that that?

Reply:
I know. Alright, perhaps we could make a note on this in the guidelines later.